internal/idutils/external_id.go

```diff
@@ -51,8 +52,8 @@ func GenerateExternalIDForSettingsObject(c coordinate.Coordinate) (string, error
 
 type ExternalIDGenerator func(coordinate.Coordinate) (string, error)
 
-// GenerateExternalIDForDocument generates an external ID for a document configuration. It is under 50 characters long and uses at most only "a-z", "A-Z", "0-9" and "-".
-func GenerateExternalIDForDocument(c coordinate.Coordinate) (string, error) {
+// GenerateExternalID generates an external ID for a document configuration. It is under 50 characters long and uses at most only "a-z", "A-Z", "0-9" and "-".
```

Comment:
Just an idea: As the function takes the coordinate, could it absorb the function above for settings and then provide the correct externalID after examining the config type?

Reply:
I like the idea, lets align at the daily :)

Reply:
Kind of like that as well, though we do increase cohesion 🤔